From 5ce4580927ac097750d4faa4326f8d1d3456bea8 Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Tue, 30 Oct 2018 15:06:59 +0800
Subject: [PATCH 4954/5242] MLK-20124-2: arch: arm64: imx8qm: Move
 irqsteer_hdmi to hdmi dts

commit  6b9f6d74b89bf9166c9a9fa7bf82a21139763780 from
https://source.codeaurora.org/external/imx/linux-imx.git

Move irqsteer_hdmi to hdmi specific dts.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-device.dtsi      |    1 +
 .../dts/freescale/fsl-imx8qm-lpddr4-arm2-dp.dts    |    4 ++++
 .../dts/freescale/fsl-imx8qm-lpddr4-arm2-hdmi.dts  |    4 ++++
 .../boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts     |    4 ++++
 4 files changed, 13 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
index d86fc0d..6ef61a4 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
@@ -2787,6 +2787,7 @@
 							<&clk IMX8QM_HDMI_LIS_IPG_CLK>;
 		assigned-clock-rates = <675000000>, <84375000>;
 		power-domains = <&pd_hdmi>;
+		status = "disabled";
 	};
 
 	irqsteer_hdmi_rx: irqsteer@58260000 {
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-dp.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-dp.dts
index bbc51c6..c9083f7 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-dp.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-dp.dts
@@ -39,6 +39,10 @@
 	status = "disabled";
 };
 
+&irqsteer_hdmi {
+	status = "okay";
+};
+
 &hdmi {
 	compatible = "fsl,imx8qm-dp";
 	assigned-clocks = <&clk IMX8QM_HDMI_PXL_SEL>,
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-hdmi.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-hdmi.dts
index 8301c8f..7cc8944f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-hdmi.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-hdmi.dts
@@ -52,6 +52,10 @@
 	status = "disabled";
 };
 
+&irqsteer_hdmi {
+	status = "okay";
+};
+
 &hdmi {
 	compatible = "fsl,imx8qm-hdmi";
 	assigned-clocks = <&clk IMX8QM_HDMI_PXL_SEL>,
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts
index 324ff44..711b35f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts
@@ -52,6 +52,10 @@
 	status = "disabled";
 };
 
+&irqsteer_hdmi {
+	status = "okay";
+};
+
 &hdmi {
 	compatible = "fsl,imx8qm-hdmi";
 	assigned-clocks = <&clk IMX8QM_HDMI_PXL_SEL>,
-- 
1.7.9.5

